.cloudbuild/release.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -o errexit -o xtrace
+
+# setup remote cache
+curl -fsSL https://github.com/buchgr/bazel-remote/releases/download/v2.4.0/bazel-remote-2.4.0-linux-x86_64 -o bazel-remote
+echo '717a44dd526c574b0a0edda1159f5795cc1b2257db1d519280a3d7a9c5addde5 bazel-remote' | sha256sum --check
+chmod +x bazel-remote
+mkdir .logs
+./bazel-remote --max_size 8 --dir ~/.cache/bazel-remote --experimental_remote_asset_api --grpc_address 0.0.0.0:4700 --gcs_proxy.bucket $REMOTE_CACHE_GCS --gcs_proxy.use_default_credentials > .logs/bazel-remote.log 2>&1 &
+
+# install bazel 6
+apt-get install bazel-6.0.0
+ln -sf /usr/bin/bazel-6.0.0 /usr/bin/bazel
+
+# setup docker-credential-gcr
```

Comment:
nit: turned out to be unnecessary. can be removed

Reply:
nice I can use that to retrigger the GHAs.